src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+let th = [...document.querySelectorAll('thead th')];
+let tr = [...document.querySelectorAll('tbody tr')];
+
+for (let j = 0; j < th.length; j++) {
+  let count = 0;
```

Comment:
boolean is a better one
```suggestion
  let isSorted = false;
```